docs/scripting/trust_scripts/sys.upgrades.mdx

```diff
@@ -37,6 +37,12 @@ The '((%Nfull%))' parameter takes a boolean. If the value is set to ((%Vtrue%)),
 
 The '((%Ni%))' parameter takes either a number or an array of numbers.
 
+#### is_script
+
+The '((%Nis_script%))' parameter takes a boolean. If the value is set to ((%Vtrue%)), ((sys.upgrades)) returns the same output as if the script was called inside of a script.
+
+If it is false, it returns the same output as if the script was called from the CLI.
+
```

Comment:
Might be worth mentioning that omitting it causes auto-detection behavior (if called as a subscript, it behaves as if `is_script: true`, otherwise as `is_script: false`)